src/probability/martingale/centering.lean

```diff
@@ -136,6 +136,41 @@ begin
   refl,
 end
 
+-- The following two lemma demonstrates the essential uniqueness of the decomposition
+lemma martingale_part_add_ae_eq [sigma_finite_filtration μ ℱ] {f g : ℕ → Ω → E}
+  (hf : martingale f ℱ μ) (hg : adapted ℱ (λ n, g (n + 1))) (hg0 : g 0 = 0)
```

Comment:
Do you really need `g 0 = 0`?

Reply:
Right, you do because our definition of predictable_part and martingale_part puts the initial value in the martingale part. I did not think about it before, and I don't know if that's the right choice

Reply:
If we don't impose the initial condition somewhere the decomposition won't be unique right?

Reply:
Yes, but we could put the initial value in the predictable part instead of the martingale part. I don't know if there is any good argument for one or the other. Let's keep it as it is.